### DISCUSSION TITLE
"Inbox" vs "Notes", what's the difference?

### DISCUSSION THREAD

**Discussion by defulmere** [glushchenko/fsnotes#1589] [Q&A] [answered]
Th FSNotes sidebar has items for "Inbox" and "Notes":

![image](https://github.com/glushchenko/fsnotes/assets/104225/a2360d6c-878a-45da-9900-6170ae178bb1)

They both seem to show the same content when I select either of them. What's the difference between these 2 items?

**Comment by glushchenko** [comment, marked as answer]
Inbox - notes in the root folder.
Notes - all notes.

**Reply by defulmere** [reply]
Hmm, ok thanks.

**Comment by gingerbeardman** [comment]
IMHO these show their differences depending on how you use FSNotes. 

For me I never tag/sort/organise anything so Inbox and Notes are effectively the same thing. So I turn them off in View > Show in Sidebar (in fact I have no FSNotes organisational items showing in Sidebar).

If you do organise, then Inbox is generally your unsorted notes.

**Reply by amarendra** [reply]
Thank you. This was helpful. I looked around but couldn't hide it on iOS. Am I missing something? Is there a way to hide items like "Inbox", "Untagged" etc on iOS as well? (I do not use note apps in such detailed manner hence thinking of hiding them).

**Reply by gingerbeardman** [reply]
I never use the sidebar on iOS, so everything is effectively hidden. But you can tap and hold on a tab to get "View Options" that might let you do what you want.
